---
geomtwo/Vector bearing_measurement
